.github/actions/deploy-happy-stack/action.yml

```diff
@@ -38,18 +46,26 @@ runs:
         CREATE_TAG: ${{ inputs.create-tag }}
         TAG: ${{ inputs.tag }}
         ENV: ${{ inputs.env }}
+        OPERATION: ${{ inputs.operation }}
+        FLAGS: ${{ inputs.flags }}
       shell: bash
       run: |
         set -ue
         set -o pipefail
         
         IMAGE_TAG="${TAG:-sha-$GITHUB_SHA}"
         echo "Targetting Stack ${STACK_NAME}, tagging with '${IMAGE_TAG}'"
+        HAPPY_OPERATION=${OPERATION}
```

Comment:
would be simpler just use OPERATION later? any benefit in reassigning another variable to it?

Reply:
I thought we couldn't change those variable for some reason, it's been fixed